.github/workflows/test.yml

```diff
@@ -64,7 +64,7 @@ jobs:
       - name: Create a file with all the pkgs
         run: go list ./... > pkgs.txt
       - name: Split pkgs into 4 files
-        run: split -d -n l/4 pkgs.txt pkgs.txt.part.
+        run: split -d -n l/4 -e pkgs.txt pkgs.txt.part.
```

Comment:
[proposal] what does the `-e` flag do? Does it require a specific `split` version because on my machine the man page doesn't describe a `-e` flag.

```
SPLIT(1)                                                  General Commands Manual                                                 SPLIT(1)

NAME
     split – split a file into pieces

SYNOPSIS
     split -d [-l line_count] [-a suffix_length] [file [prefix]]
     split -d -b byte_count[K|k|M|m|G|g] [-a suffix_length] [file [prefix]]
     split -d -n chunk_count [-a suffix_length] [file [prefix]]
     split -d -p pattern [-a suffix_length] [file [prefix]]

DESCRIPTION
     The split utility reads the given file and breaks it up into files of 1000 lines each (if no options are specified), leaving the file
     unchanged.  If file is a single dash (‘-’) or absent, split reads from the standard input.

     The options are as follows:

     -a suffix_length
             Use suffix_length letters to form the suffix of the file name.

     -b byte_count[K|k|M|m|G|g]
             Create split files byte_count bytes in length.  If k or K is appended to the number, the file is split into byte_count
             kilobyte pieces.  If m or M is appended to the number, the file is split into byte_count megabyte pieces.  If g or G is
             appended to the number, the file is split into byte_count gigabyte pieces.

     -d      Use a numeric suffix instead of a alphabetic suffix.

     -l line_count
             Create split files line_count lines in length.

     -n chunk_count
             Split file into chunk_count smaller files.  The first n - 1 files will be of size (size of file / chunk_count ) and the last
             file will contain the remaining bytes.

     -p pattern
             The file is split whenever an input line matches pattern, which is interpreted as an extended regular expression.  The
             matching line will be the first line of the next output file.  This option is incompatible with the -b and -l options.

     If additional arguments are specified, the first is used as the name of the input file which is to be split.  If a second additional
```

Update: https://man7.org/linux/man-pages/man1/split.1.html describes

>  -e, --elide-empty-files
>              do not generate empty output files with '-n'

if that's accurate then proposal to use

```suggestion
        run: split -d -n l/4 --elide-empty-files pkgs.txt pkgs.txt.part.
```

Motivation: https://changelog.com/posts/use-long-flags-when-scripting

Reply:
@rahulghangas why was this resolved with `-e` still present? I think it was commited and then reverted.

Reply:
Sorry, forgot to push the change I think